tools/backups2datalad/__main__.py

```diff
@@ -123,6 +138,8 @@ async def update_from_backup(
     asset_filter: Optional[re.Pattern[str]],
     force: Optional[str],
     workers: Optional[int],
+    verify_timestamps: Optional[bool],
+    gc_assets: Optional[bool],
```

Comment:
please add unittest runs with verify_timestamps = True (could be "dummy" in that there would be no changes) and `gc_assets = True` -- nothing should be GCed in that test case I assume